docs/topics/db/sql.txt

```diff
@@ -212,11 +212,43 @@ of people with their ages calculated by the database:
     Jane is 42.
     ...
 
-You can often avoid using raw SQL to compute annotations by instead using a
-:ref:`Func() expression <func-expressions>`.
+You can often avoid using raw SQL to compute annotations by using a
+:ref:`Func() expression <func-expressions>`, or use :class:`~django.db.models.expressions.RawSQL`
+for more control when needed.
 
 __ https://www.postgresql.org/docs/current/functions-datetime.html
 
+Using RawSQL in ORM expressions
+================================
+
+Django provides a number of built-in expressions and functions for use in queries.
+However, when these are not sufficient, you can use :class:`~django.db.models.expressions.RawSQL`
+to include custom SQL fragments inside annotations or filters.
```

Comment:
This intro to the topic doesn't really place `RawSQL` within the context of the surrounding page. It should highlight how `RawSQL` differs to the other options.